### PR TITLE
I4652 - disbales submit button if no name

### PR DESCRIPTION
**Pull request by tsl143**
Fixes: #4652 

Before:
screen shot 2018-04-01 at 6 29 26 pm

After: 
screen shot 2018-04-02 at 8 09 21 am

### CODE REVIEW

File: src/amo/components/CollectionManager/index.js [hunk 1]
```diff
@@ -169,7 +170,7 @@ export class CollectionManagerBase extends React.Component<Props, State> {
     // https://github.com/mozilla/addons-frontend/issues/4635
     // The collectionUpdates state will handle this but it needs
     // to be hooked up the saga.
-    const formIsDisabled = !collection;
+    const formIsDisabled = (!collection || !name.trim());
```
Comment (tofumatt) [inline src/amo/components/CollectionManager/index.js]: I guess the tests pass with this method but I would have thought we needed to test for `.length` here, as `''.trim()` is still truthy.

**Comment by tsl143** [comment]
I assumed, we need to replicate same behavior for cancel as submit. I will change this :)

File: src/amo/components/CollectionManager/index.js (hunk 1 above)
Comment (tsl143) [reply src/amo/components/CollectionManager/index.js]: `''.trim()` is falsey, isnt it?

Comment (tsl143) [reply src/amo/components/CollectionManager/index.js]: image

**Comment by tsl143** [comment]
@tofumatt , updated :)

File: tests/unit/amo/components/TestCollectionManager.js [hunk 2]
```diff
@@ -256,6 +256,16 @@ describe(__filename, () => {
     }));
   });
 
+  it('disables submit button when the name is blank', () => {
+    const root = render();
+
+    // Enter in a blank collection name.
+    typeInput({ root, name: 'name', text: '' });
+
+    expect(root.find('.CollectionManager-submit'))
```
Comment (tofumatt) [inline tests/unit/amo/components/TestCollectionManager.js]: Might be nice to add an extra `expect` statement here that makes sure the cancel button *isn't* disabled.

**Comment by tsl143** [comment]
@tofumatt updated test case.

**Pull request merged**